packages/css/src/components/page-menu/page-menu.scss

```diff
@@ -28,14 +40,19 @@
 }
 
 .ams-page-menu--align-end {
-  justify-content: end;
+  justify-content: flex-end;
+  margin-inline-start: auto;
 }
 
-.ams-page-menu--no-wrap {
-  flex-wrap: nowrap;
+.ams-page-menu__item {
+  &--secondary {
+    @media screen and (not (min-width: $ams-breakpoint-medium)) {
+      display: none;
+    }
+  }
 }
 
-@mixin page-menu-item {
+@mixin ams-page-menu-link-or-button {
```

Comment:
I understand you want to improve the semantics but you've only renamed this mixin, the tokens are still called `-ams-page-menu-item`, so it increases the confusion for me. Ie. Are we targetting the li or the button/link? And what if you ever add a username dropdown to the mix?

Reply:
Hmm, yes, the token names… The links and buttons share many styles, so keeping the mixin seemed worthwhile. Items (`li`) have always been the parent element of both, so it was a bit confusing already.

We could go with `ams-page-menu-element` to target links, buttons, and future elements that look the same?

Reply:
Sure.